.github/workflows/ci.yml

```diff
@@ -167,14 +167,20 @@ jobs:
         id: configure-command
         run: |
           # Common CMake arguments
-          args=( -S . -B ${{ matrix.build-dir }} -DCMAKE_BUILD_TYPE=${{ matrix.build-type }} )
+          args=( -S . -B ${{ matrix.build-dir }} )
 
           # Generator's specific stuff
           # We don't add generator here to avoid splitting it before
```

Comment:
Not related to this change, but I realized that I don't understand what this comment means. Can you clarify it?

Reply:
Consider the following example:
```bash
args=(-DCMAKE_BUILD_TYPE=Debug -G "Unix Makefiles")

str_args=$(echo "${args[@]}")
arr_args=($str_args)

for i in "${arr_args[@]}"; do echo "ARG: $i"; done
```

Expected output:
```
ARG: -DCMAKE_BUILD_TYPE=Debug
ARG: -G
ARG: Unix Makefiles
```

Actual output:
```
ARG: -DCMAKE_BUILD_TYPE=Debug
ARG: -G
ARG: Unix
ARG: Makefiles
```

This is why we don't add generator in 'Prepare Configure Command' step

Reply:
Ah, I see, thanks! I couldn't understand that it refers to word split. Can you reword the comment as "We don't add generator here to avoid word split when expanding array elements, e.g. `Unix Makefiles` passed as two arguments".

Reply:
(Not necessarily in this PR.)

Reply:
Could you please take a look https://github.com/skvadrik/re2c/pull/332/commits/5e7ac621cdfa72f65ceea75e38b7ac711fff9097